funding.json

```diff
@@ -1,5 +1,15 @@
 {
+    "Optimism": {
+        "op-mainnet": {
+            "ownedBy": "0xc44F30Be3eBBEfdDBB5a85168710b4f0e18f4Ff0"
+        }
+    },
+    "drips": {
+        "ethereum": {
+            "ownedBy": "0xc44F30Be3eBBEfdDBB5a85168710b4f0e18f4Ff0"
+        }
+    },
```

Comment:
unsure why github is flagging these as changes but would be worth double checking that these addresses are accurate